Library/Homebrew/cmd/style.rb

```diff
@@ -23,9 +19,11 @@
 module Homebrew
   def style
     target = if ARGV.named.empty?
-      [HOMEBREW_LIBRARY]
+      [HOMEBREW_LIBRARY/"Homebrew"]
```

Comment:
You should use `HOMEBREW_LIBRARY_PATH` here. It might matter in tests (now or possibly later) when `HOMEBREW_LIBRARY_PATH != HOMEBREW_LIBRARY/"Homebrew"`.